react-responsive-modal/package.json

```diff
@@ -55,13 +55,12 @@
     }
   ],
   "dependencies": {
-    "@bedrock-layout/use-forwarded-ref": "^1.1.4",
```

Comment:
now that the latest version support react 18 could you please just upgrade the lib?

Reply:
done